frontend/src/metabase-lib/v1/expressions/config.ts

```diff
@@ -64,6 +64,18 @@ export const MBQL_CLAUSES: MBQLClauseMap = {
     args: ["number"],
     requiresFeature: "standard-deviation-aggregations",
   },
+  offset: {
+    displayName: `Offset`,
+    type: "aggregation",
+    args: ["any", "number"],
```

Comment:
This can accept basically any sort of expression, we didn't have a compatible type so I added `any`